code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm

```diff
@@ -117,7 +117,7 @@
 	for(var/mob/M in range(10,src))
 		if(M.client)
 			shake_camera(M, 4, 3)
-	visible_message(SPAN_DANGER(pick("Prepare to die!", "JUSTICE", "Run!")))
+	visible_message(SPAN_DANGER(pick("JUDGING...", "THE HAND OF FATE DECIDES", "THE SKIES THE SKIES THE SKIES THE SKIES THE SKIES THE SKIES THE SKIES THE SKIES THE SKIES THE SKIES THE SKIES THE SKIES THE SKIES THE SKIES THE SKIES THE SKIES THE SKIES THE SKIES THE SKIES THE SKIES THE SKIES THE SKIES ")))
```

Comment:
I'm worried "THE SKIES" might spam the chat a little too much. Is one of these only spoken once or multiple times?

Reply:
These are messages that the big bad boy says to choreograph when he's about to use a big bad attack, hence why the length on all of them are increased. It also doesn't spam the chat as much as you'd suspect! Can still reduce it if you want, but I don't think I need to.